### PR TITLE
Override artifact output with BIN_PATH

### PR DESCRIPTION
**Pull request by ashcrow**

### CODE REVIEW

File: build_releases [hunk 1]
```diff
@@ -21,14 +21,17 @@ fi
 
 eval $(go env)
 
-export GOBIN=${PWD}/bin/releases
+if [ -z ${BIN_PATH+a} ]; then
+	export BIN_PATH=${PWD}/bin/${GOARCH}
```
Comment (arithx) [inline build_releases]: Previously this was output in `${PWD}/bin/releases`

Comment (ashcrow) [reply build_releases]: good catch

**Comment by ashcrow** [comment]
Updated

**Comment by dgonyeo** [comment]
The binary names for the releases already have the architecture in their names. With this change, the arm64 linux binary is now `ignition-validate-aarch64-unknown-linux-gnu.arm64`. Correct me if I'm wrong, but the `aarch64` and `arm64` seem redundant.

**Comment by ashcrow** [comment]
Updated

**Pull request merged**